ns1/resource_record.go

```diff
@@ -473,6 +504,28 @@ func metaDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 		return len(compareMap) == 0
 	}
 
+	if metaDiffSuppressUp(k, old, new, d) {
+		return true
+	}
+
+	return false
+}
+
+func metaDiffSuppressUp(k, old, new string, d *schema.ResourceData) bool {
+	if strings.HasSuffix(k, "up") {
+		if trueString(new) != trueString(old) {
```

Comment:
`trueString()` looks really similar to `strconv.Parsebool()`.  Could this func be rewritten as:
```
if strings.HasSuffix(k, "up") {
    newB, _ := strconv.ParseBool(new)
    oldB, _ := strconv.ParseBool(old)
    if newB == oldB {
			return true
    }
}
return false
```

Reply:
oh yeah, good call